RealifeTech-CoreSDK.xcodeproj/project.pbxproj

```diff
@@ -7,6 +7,11 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		0909B12325FF4AE40026A9B5 /* payment_getMyPaymentSources.graphql.swift in Sources */ = {isa = PBXBuildFile; fileRef = 0909B12225FF4AE40026A9B5 /* payment_getMyPaymentSources.graphql.swift */; };
```

Comment:
Could you rename these graphql files with the prefix `sell_` as they belong to the Sell interface on Frame/Mobile SDK?

Reply:
No problem. Will update this. Thank you!